scex-core/src/main/scala/com/avsystem/scex/presentation/MathAttributes.scala

```diff
@@ -0,0 +1,120 @@
+      m.addExact(_: Long, _: Long) --> Attributes(documentation = "Returns the sum of `x` and `y` arguments.")
+
+      m.asin _ --> Attributes(documentation = "Returns the arc sine of the `x` argument.")
+      m.atan _ --> Attributes(documentation = "Returns the arc tangent of the `x` argument.")
+      m.atan2 _ --> Attributes(documentation = "Returns the rectangular coordinates (x, y) converted to polar (r, theta).")
+
+      m.cbrt _ --> Attributes(documentation = "Returns the cube root of the `x` argument.")
+      m.ceil _ --> Attributes(documentation = "Returns the smallest double value that is greater than or equal to the `x` argument and is equal to a mathematical integer.")
+
+      m.copySign(_: Double, _: Double) --> Attributes(documentation = "Returns the first argument `magnitude` with the sign of the second argument `sign`.")
+      m.copySign(_: Float, _: Float) --> Attributes(documentation = "Returns the first argument `magnitude` with the sign of the second argument `sign`.")
+
+      m.cos _ --> Attributes(documentation = "Returns the cosine of the `x` argument.")
+      m.cosh _ --> Attributes(documentation = "Returns the hyperbolic cosine of the `x` argument.")
+
+      m.decrementExact(_: Int) --> Attributes(documentation = "Returns the `x` argument decremented by one.")
+      m.decrementExact(_: Long) --> Attributes(documentation = "Returns the `x` argument decremented by one.")
+
+      m.exp _ --> Attributes(documentation = "Returns the Euler's number raised to the power of the `x` argument.")
+      m.expm1 _ --> Attributes(documentation = "Return the Euler's number raised to the power of the `x` argument with one subtracted from it.")
+
+      m.floor _ --> Attributes(documentation = "Returns the largest integer value less than or equal to the `x` argument.")
+      m.floorDiv(_: Int, _:Int) --> Attributes(documentation = "Returns the largest integer value less than or equal to the algebraic quotient, " +
+        "where `x` is the dividend and `y` is the divisor.")
+      m.floorDiv(_: Long, _:Long) --> Attributes(documentation = "Returns the largest integer value less than or equal to the algebraic quotient, " +
+        "where `x` is the dividend and `y` is the divisor.")
+      m.floorMod(_: Int, _:Int) --> Attributes(documentation = "Returns the largest integer value less than or equal to the algebraic quotient, " +
+        "where `x` is the dividend and `y` is the divisor.")
+      m.floorMod(_: Long, _:Long) --> Attributes(documentation = "Returns the largest integer value less than or equal to the algebraic quotient, " +
+        "where `x` is the dividend and `y` is the divisor.")
+
+      m.getExponent(_: Double) --> Attributes(documentation = "Returns the size of unit in the last place of the `d` argument.")
+      m.getExponent(_: Float) --> Attributes(documentation = "Returns the size of unit in the last place of the `f` argument.")
```

Comment:
The javadocs on the inner method say that it returns 'an unbiased exponent', why the change in the description?